### PR TITLE
add ability to read the content for the srcinfo from a variable

### LINKED ISSUE #4: [Feature Request] Reading a string not a file
Would be nice if I could fetch the SRCINFO via urllib.request and parse it directly with python-pkgbuild and don't have to write it out to the file system.

### PR DESCRIPTION
**Pull request by RubenKelevra**
Fixes #4

### CODE REVIEW

File: pkgbuild.py [hunk 1]
```diff
@@ -8,14 +8,20 @@
 
 
 class SRCINFO:
-    def __init__(self, filepath):
+    def __init__(self, filepath="", content=""):
         """
         Parses and holds data of a .SRCINFO file.
         ;param filepath: Path to a .SRCINFO file
```
Comment (z3ntu) [inline pkgbuild.py:14]: please add `content` to the docstring

File: pkgbuild.py [hunk 2]
```diff
@@ -8,14 +8,20 @@
...
         Access the data with the object 'content'
         """
         return_dict = defaultdict(list)
-        for line in open(filepath, 'r'):
+        
+        if filepath != "":
```
Comment (z3ntu) [inline pkgbuild.py]: `if not filepath:` should also work and looks nicer

**Review by z3ntu — APPROVED** [review]
fixed some stuff, lgtm now!

**Comment by z3ntu** [comment]
Glad you find it useful!

**Comment by RubenKelevra** [comment]
> Glad you find it useful!

If you're curious, I wanna use it on my [pacman.store](https://github.com/RubenKelevra/pacman.store#pacmanstore) project :)

**Pull request merged**